components/style/properties/gecko.mako.rs

```diff
@@ -497,6 +497,7 @@ impl Debug for ${style_struct.gecko_struct_name} {
         "LengthOrPercentage": impl_style_coord,
         "LengthOrPercentageOrAuto": impl_style_coord,
         "LengthOrPercentageOrNone": impl_style_coord,
+        "MinLength": impl_style_coord,
```

Comment:
@Manishearth told me that I couldn't use `impl_style_coord` any more. Was that meant to be descriptive, or prescriptive? I did have something that compiles so I think I was able to use it, but maybe you meant that `impl_style_coord` isn't meant for this use case or something like that. If I'm not supposed to use `impl_style_coord`, what do I do to cover `min-height` and `min-width`? (Just do it twice? Share a method?)